content/doc/book/getting-started/index.adoc

```diff
@@ -9,9 +9,18 @@ layout: chapter
 :notitle:
 
 = Getting Started with Jenkins
+This chapter intended for new users unfamiliar with Jenkins or those with experience
+with Jenkins versions more than a year old.
```

Comment:
"more than a year old" seems kind of arbitrary and specific. Perhaps just "or those without experience with recent versions of Jenkins"